### PR TITLE
ADA Compliance Dec 2024

### PR DESCRIPTION
**Pull request by internetisaiah**
### Why are you making this change? (required)

[These are the top pages visited on Braze Docs](https://docs.google.com/spreadsheets/d/1vmQIXACCU4exnHCMHT9HkJQauJg5ykcAELglzz7nUO0/edit?gid=0#gid=0), so we have decided to ensure that all their images have alt text and any existing alt text does not use directional language.

_*Original PR had an issue with the commits._

### Related PRs, issues, or features (optional)

- [BD-3620](https://jira.braze.com/browse/BD-3620)
- Fixes #8503

### Feature release date (optional)

- N/A

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/custom_data/custom_events.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: "This article describes custom events and properties, segmentation,
 search_rank: 2
 ---
 
-# [![Braze Learning course]({% image_buster /assets/img/bl_icon3.png %})](https://learning.braze.com/custom-events-and-attributes){: style="float:right;width:120px;border:0;" class="noimgborder"}Custom events
+# [![Braze Learning course icon.]({% image_buster /assets/img/bl_icon3.png %})](https://learning.braze.com/custom-events-and-attributes){: style="float:right;width:120px;border:0;" class="noimgborder"}Custom events
```
Comment (internetisaiah) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_events.md:10]:
```suggestion
# [![Braze Learning course]({% image_buster /assets/img/bl_icon3.png %})](https://learning.braze.com/custom-events-and-attributes){: style="float:right;width:120px;border:0;" class="noimgborder"}Custom events
```

File: _docs/_user_guide/message_building_by_channel/content_cards/about.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ channel:
 search_rank: 4
 ---
 
-# [![Braze Learning course]({% image_buster /assets/img/bl_icon3.png %})](https://learning.braze.com/content-cards){: style="float:right;width:120px;border:0;" class="noimgborder"} About Content Cards
+# [![Braze Learning course icon.]({% image_buster /assets/img/bl_icon3.png %})](https://learning.braze.com/content-cards){: style="float:right;width:120px;border:0;" class="noimgborder"} About Content Cards
```
Comment (internetisaiah) [inline _docs/_user_guide/message_building_by_channel/content_cards/about.md:11]:
```suggestion
# [![Braze Learning course]({% image_buster /assets/img/bl_icon3.png %})](https://learning.braze.com/content-cards){: style="float:right;width:120px;border:0;" class="noimgborder"} About Content Cards
```

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/making_an_api_call.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ description: "This reference article covers how to make a Connected Content API
 search_rank: 2
 ---
 
-# [![Braze Learning course]({% image_buster /assets/img/bl_icon3.png %})](https://learning.braze.com/connected-content){: style="float:right;width:120px;border:0;" class="noimgborder"}Making an API call
+# [![Braze Learning course icon.]({% image_buster /assets/img/bl_icon3.png %})](https://learning.braze.com/connected-content){: style="float:right;width:120px;border:0;" class="noimgborder"}Making an API call
```
Comment (internetisaiah) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/making_an_api_call.md:9]:
```suggestion
# [![Braze Learning course]({% image_buster /assets/img/bl_icon3.png %})](https://learning.braze.com/connected-content){: style="float:right;width:120px;border:0;" class="noimgborder"}Making an API call
```

**Comment by internetisaiah** [comment]
All of @bre-fitzgerald 's edits have been migrated over as well, so this would should only need a quick review before merging 👍🏽

File: _docs/_user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas.md [hunk 4]
```diff
@@ -173,7 +173,7 @@ Note that:
 
 To view additional statistics, such as the average lifetime revenue for targeted users, select **Show Additional Statistics**.
 
-![Target Population breakdown with option to calculate exact statistics.][2]
+![Target Population breakdown with option to cacluate exact statistics.][2]
```
Comment (josh-mccrowell-braze) [inline _docs/_user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas.md:176]:
```suggestion
![Target Population breakdown with option to calculate exact statistics.][2]
```

**Review by josh-mccrowell-braze — APPROVED** [review]

**Pull request merged**